backend/src/main/validation/PromotionQueryValidation.ts

```diff
@@ -16,7 +16,11 @@ export class PromotionQueryValidation {
      * */
     discountValue: Joi.number().positive().precision(2),
     promotionType: Joi.string().valid(...Object.values(PromotionType)),
-    cuisine: Joi.string().valid(...Object.values(CuisineType)),
+    /** request.query needs to follow format in order to be automatically converted to array - https://stackoverflow.com/a/33086861 */
+    cuisine: Joi.alternatives(
+      Joi.string().valid(...Object.values(CuisineType)),
+      Joi.array().items(Joi.string().valid(...Object.values(CuisineType)))
+    ),
```

Comment:
@andrewzulaybar is it difficult to format the request query in the image? I got this from here https://stackoverflow.com/a/33086861. 
![image](https://user-images.githubusercontent.com/49849754/103491645-71a85a80-4dda-11eb-9cc0-5c9de5258a4f.png)

I found this an easier solution because if I made the request query something like this I would need to use `JSON.parse` and it got complicated with Joi
![image](https://user-images.githubusercontent.com/49849754/103491679-a0263580-4dda-11eb-8afe-7f14fe43f0e4.png)

Reply:
I don't think that'd be too hard, so I can change it!

Reply:
Sweet thanks!